### ISSUE TITLE
Better Notes Editor not working

### ISSUE THREAD

**Issue by Darthagnon** [LonMcGregor/VivaldiMods#22]
The Better Notes Editor doesn't seem to work anymore (v2.5 snapshot). I placed all 6x files in ...resources\vivaldi\mods, and added the needed paths to browser.html. As far as I can tell, from betterNotes.js, this is the correct path, as:` const EDITOR_URI = "chrome-extension://mpognobbkildjkofajifpdfhcoklimli/mods/betterNotesEditor.html";` seems to specify this folder, not the "user_mods" folder you refer to in your readmes (NB. I have tried placing betterNotesEditor.html in a folder @ ...resources\vivaldi\user_mods, but this made no difference).

So it looks like it might be broken? Unless I'm being really dense...

**Comment by LonMcGregor** [comment]
I remember changing the path of the files to just "mods", I'm not sure if that would impact too much.

I'll take a look at this.

**Comment by Darthagnon** [comment]
I just tried installing it again. It works, the icon looked like a folder icon. I'm an idiot; my apologies.

**Comment by Darthagnon** [comment]
Quick question, is there a way to move the controls at the bottom of the page, to the top? I use a custom CSS for the status bar that turns it into a floating popup overlay, and it blocks access to the controls.

**Comment by LonMcGregor** [comment]
Yeah, I suppose the icon isn't very clear.

For the status bar controls, you could try editing the HTML file and put the `footer` just after the `header`.

**Comment by Darthagnon** [comment]
Thank you so much for your responses!

Icon is ok, imo, now that I know what I'm looking for. at first, I just thought it was a folder icon, and I was expecting an icon like the favicon PNG you included to be the icon in the panel menu.

Edited the HTML file, looks pretty sweet, now - please find it as attachment (rename from png to html) (you might prefer to the default?)
![betterNotesEditor html](https://user-images.githubusercontent.com/20792748/56913863-9b0fcf80-6aaa-11e9-9dd6-44b9e495cb1c.png)

Another quick question, (sorry!): what elements control the font/button colour? I'm using some custom colouring, and the buttons are unreadable (and I couldn't find anything referring to Vivaldi colour variables, other than a couple of refs in betterNotesEditor.css)
![2019-04-29 18_17_57-Vivaldi Notes - Vivaldi](https://user-images.githubusercontent.com/20792748/56914097-1f625280-6aab-11e9-88e1-709a942881a9.png)

**Comment by LonMcGregor** [comment]
Good catch on the buttons - I don't use a dark theme very often, so I missed them. I've pushed an update which corrects the problem.

Commit 1fb76994e566e
